llvm/include/llvm/TargetParser/AArch64TargetParser.h

```diff
@@ -116,12 +116,18 @@ using ExtensionBitset = Bitset<AEK_NUM_EXTENSIONS>;
 // SubtargetFeature which may represent either an actual extension or some
 // internal LLVM property.
 struct ExtensionInfo {
-  StringRef Name;                 // Human readable name, e.g. "profile".
+  StringRef UserVisibleName;      // Human readable name used in -march, -cpu
```

Comment:
Please could you make the names here consistent with tablegen, i.e. either pick `UserVisibleName` or `MArchName`.